pyvista/plotting/_utilities.py

```diff
@@ -0,0 +1,51 @@
+"""Utilities for plotting."""
+from typing import Tuple
+
+import numpy as np
+
+
+def view_vectors(view: str, negative: bool = False) -> Tuple[np.ndarray, np.ndarray]:
+    """Given a plane to view, return vectors for setting up camera.
+
+    Parameters
+    ----------
+    view : {'xy', 'yx', 'xz', 'zx', 'yz', 'zy'}
```

Comment:
This is a numpydoc supported format, but I'm not sure if classifies it as conventional.  I tend to like this format since it is immediately obvious that it isn't _any_ string but only one from a set of six possibilities.  But I'm fine if the PyVista preference is to only use the type.

Reply:
If it's supported by numpydoc then it's good enough for me, thanks. I'll let this unresolved for now to let others chime in.